### PR TITLE
Pull Sonarr commit 'New: Allow user to choose whether delay profile should apply to release of the highest enabled quality'

### LINKED ISSUE #5999: Pull Sonarr commit 'New: Allow user to choose whether delay profile should apply to release of the highest enabled quality'
Pull https://github.com/Sonarr/Sonarr/commit/d668e923af83ab7f1589d947fec9f40a3919e0b8

```
git cherry-pick -ex d668e923af83ab7f1589d947fec9f40a3919e0b8
```

This commit has 2 conflict(s)

You can ask me to `close all`, `open pr here` or `open pr everywhere`

### PR DESCRIPTION
**Pull request by RobinDadswell**
(cherry picked from commit d668e923af83ab7f1589d947fec9f40a3919e0b8)

Database Migration 194

* Fixes #5999

### CODE REVIEW

**Comment by bakerboy448** [comment]
Note: No wiki updates needed; exists already (common template w/ sonarr)

File: src/NzbDrone.Core.Test/DecisionEngineTests/RssSync/DelaySpecificationFixture.cs [hunk 1]
```diff
@@ -104,10 +104,22 @@ public void should_be_true_when_profile_does_not_have_a_delay()
...
 
         [Test]
-        public void should_be_true_when_quality_is_last_allowed_in_profile()
+        public void should_be_false_when_quality_is_last_allowed_in_profile_and_bypass_disabled()
         {
             _remoteMovie.ParsedMovieInfo.Quality = new QualityModel(Quality.Bluray720p);
 
+            _remoteMovie.Release.PublishDate = DateTime.UtcNow;
+
+            Subject.IsSatisfiedBy(_remoteMovie, null).Accepted.Should().BeFalse();
```
Comment (RobinDadswell) [inline src/NzbDrone.Core.Test/DecisionEngineTests/RssSync/DelaySpecificationFixture.cs:113]: This was my fix to the test failing - Taloth never got back to me but as the test was name "should be false" I assumed this should be false and not true as it is in the Sonarr Cherry-Pick :)

File: frontend/src/Settings/Profiles/Delay/EditDelayProfileModalContent.js [hunk 2]
```diff
@@ -110,6 +111,20 @@ function EditDelayProfileModalContent(props) {
                   </FormGroup>
               }
 
+              {
+                <FormGroup>
+                  <FormLabel>Bypass if Highest Quality</FormLabel>
```
Comment (Qstick) [inline frontend/src/Settings/Profiles/Delay/EditDelayProfileModalContent.js:116]: Translate

File: frontend/src/Settings/Profiles/Delay/EditDelayProfileModalContent.js [hunk 3]
```diff
@@ -110,6 +111,20 @@ function EditDelayProfileModalContent(props) {
...
+
+                  <FormInputGroup
+                    type={inputTypes.CHECK}
+                    name="bypassIfHighestQuality"
+                    {...bypassIfHighestQuality}
+                    helpText="Bypass delay when release has the highest enabled quality in the quality profile"
```
Comment (Qstick) [inline frontend/src/Settings/Profiles/Delay/EditDelayProfileModalContent.js:122]: Translate

**Review by Qstick — CHANGES_REQUESTED** [review]
Just the two translates

**Review by Qstick — APPROVED** [review]

**Pull request merged**